### PR TITLE
Monkeys now do interesting stuff on spawning with random numbers

### PR DESCRIPTION
**Pull request by ghost**
:cl: McBawbaggings
add: Monkeys will now have unique qualities and items when they get certain random numbers for their names.
/:cl:

If anyone has extra stuff they want me to add, just say.

Currently features:
2 scoops
baneposting
james bond
revolver ocelot
17% = 100%
beer
death note
0.5x a presses
butts
gandhi
$322
weed
death
rey mysterio
satan
gambling
emergency medical services

### CODE REVIEW

**Comment by tortellinitony** [comment]
good idea

**Comment by j-mie** [comment]
+1

**Comment by ghost** [comment]
I've finished writing most of my ideas.
I've double tested all scenarios to ensure items spawn as expected.
Some of the inhands aren't showing but that's just inhands stuff anyway.

I've also ensured the naming functionality equips the items as they spawn by spamming monkey spawns and checking and seeing it happen.

File: hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			name = "Solo"
+		if(420)
+			equip_to_slot_or_del(new /obj/item/clothing/mask/cigarette/rollie/trippy(src), slot_wear_mask)
+			equip_to_slot_or_del(new /obj/item/clothing/head/beanie/green(src), slot_head)
+		if(444)
+			death()
+		if(666)
+			fire_stacks = 6
+			IgniteMob()
+		if(777) /* bugged */
```
Comment (KayeArray) [inline hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm]: how so?

File: hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+			cigar.light()
+			equip_to_slot_or_del(cigar, slot_wear_mask)
+			put_in_hands_or_del(martini)
+			put_in_hands_or_del(new /obj/item/toy/gun(src))
+		if(12)
+			put_in_hands_or_del(new /obj/item/toy/gun(src))
+			put_in_hands_or_del(new /obj/item/toy/gun(src))
+			equip_to_slot_or_del(new /obj/item/clothing/head/beret(src), slot_head)
+		if(17)
+			if(prob(17))
```
Comment (KayeArray) [inline hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm:27]: ??

File: hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm]: my bad, i used comments to signal out which ones weren't working during testing and i forgot to remove that one

File: hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm:27]: https://dota2.gamepedia.com/Memes#17.25

Comment (KayeArray) [reply hippiestation/code/modules/mob/living/carbon/monkey/monkey.dm:27]: eww

**Comment by ghost** [comment]
small but interesting
+1

**Comment by DinduStuffin** [comment]
+1

**Review by GuyonBroadway — APPROVED** [review]

**Pull request merged**